examples/arduino/z_sub.ino

```diff
@@ -111,4 +111,4 @@ void setup() {
     Serial.println("ERROR: Zenoh pico was compiled without Z_FEATURE_SUBSCRIPTION but this example requires it.");
```

Comment:
I agree but clang-format in this current configuration doesn't it seems. I'll look into it.

Reply:
OK so basically clang-format tries to put functions in a single line when possible, here it doesn't because it's the only one going over 120 char in width.

There is no values for the field `AllowShortFunctionsOnASingleLine` that's interesting for our case, it's basically we remove all one-line functions or we're forced to have them when possible.

Reply:
As a compromise, I added a return statement in all functions.